code/modules/mob/living/living.dm

```diff
@@ -187,8 +187,8 @@
 
 			var/mob/living/L = M
 
-			var/self_points = FLOOR((STAEND + STASTR + mind.get_skill_level(/datum/skill/misc/athletics))/2, 1) //Constitution had too many quirk bonuses associated.
-			var/target_points = FLOOR((L.STAEND + L.STASTR + L.mind.get_skill_level(/datum/skill/misc/athletics))/2, 1)
+			var/self_points = FLOOR((STACON + STASTR + mind.get_skill_level(/datum/skill/misc/athletics))/2, 1) //Constitution had too many quirk bonuses associated.
```

Comment:
This was changed to STAEND to discourage CON baiting. If you want the high CON classes to be better at charging, consider giving those classes the Charger trait. This would makes sense mostly for the barbarian, but possibly the unarmed-focused Fighter subclass too.